src/cordova/setup.action.mjs

```diff
@@ -58,6 +59,8 @@ export async function main(...parameters) {
     );
   }
 
+  await rmfr(`platforms/${platform}`);
```

Comment:
Why are you deleting the platform right after you add?
This will break on macOS, since the prepare below will try to pull the platform from head, which is broken.